README.md

```diff
@@ -1,5 +1,9 @@
 # Keycloak Event Listener for Temporal
 
+![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/InfinityFlowApp/keycloak-event-listener-temporal/release.yml)

```

Comment:
The links just open up a new tab with the image.
Should they go somewhere?
Like the latest release page and he license file?

Reply:
Idea was just to show the quick info, but GitHub seems to differently displays markdown than in my IDE.

Reply:
builds -> actions
release -> releases
license -> license tab